sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonSuite.scala

```diff
@@ -3820,6 +3820,21 @@ abstract class JsonSuite
       }
     }
   }
+
```

Comment:
@HyukjinKwon 
In each `***Suite`, I added `UT` to verify this issue: when `specify the schema`, will return `empty dataset.`